preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -400,6 +406,46 @@ def concatenate(
                 warnings=warnings,
             )
 
+    @staticmethod
+    def check_authors(
+        input_data: InputMetadata, output_field: str, args: FunctionArgs = None
+    ) -> ProcessingResult:
+        authors = input_data["authors"]
+
+        if not authors:
+            return ProcessingResult(
+                datum=None,
+                warnings=[],
+                errors=[],
+            )
+        warnings: list[ProcessingAnnotation] = []
+        errors: list[ProcessingAnnotation] = []
+        if valid_authors(authors):
+            return ProcessingResult(
+                datum=authors,
+                warnings=warnings,
+                errors=errors,
+            )
+
+        error_message = (
+            f"The authors list '{authors}' are not in a recognized format. Please ensure that "
+            "authors are separated by semi-colons, and that each author’s last name and first "
+            "name are separated by a comma. "
+            "For example: 'lastname, firstname; lastname, firstname'."
```

Comment:
```suggestion
            "For example: 'lastname, firstname; lastname, firstname'."
```

We might want to add a more real example in particular one with multiple first names and/or initials

Reply:
Also we might want to mention these edge cases in the docs and also here in the error message, otherwise it can be confusing

```
    "Xi,;",
```

I.e. last name only must still have comma, and first name only is not allowed.

Reply:
Ah yes this is lacking the end ; good catch!